### ISSUE TITLE
More clarity on the warmup step before pretraining

### ISSUE THREAD

**Issue by akshat311** [google-research/pix2struct#42]
I was unable to determine the specific number of distinct images utilized in the warmup phase from the paper. Also, although it's noted that training took place over 30k steps, the batch size for this phase isn't clearly stated. Can you offer any insights on this and any other hyperparameter that can be essential?
